microsoft/PowerToys
# [FancyZones] Move FancyZones out of the runner process

## Summary of the Pull Request

**What is this about:**

* Renamed projects
* Start and terminate separate FZ process from the runner. 
* Added global exception handler
* Changed settings saving
* Added the event to launch the Editor from the Settings.
* Updated installer and pipeline
* Updated settings telemetry

**What is include in the PR:** 

**How does someone test / validate:** 

* Install PowerToys, verify that FZ works.
* Go through the FZ check list: https://github.com/microsoft/PowerToys/blob/releaseChecklist/doc/releases/tests-checklist-template.md#fancyzones
* Exit PT, verify that `PowerToys.FancyZones.exe` stopped.

## Quality Checklist

- [x] **Linked issue:** #11077

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
